inst/tests/tests.Rraw

```diff
@@ -14959,12 +14959,24 @@ test(2041.1, DT[, median(date), by=g], data.table(g=c("a","b"), V1=as.Date(c("20
 test(2041.2, DT[, median(time), by=g], DT[c(2,5),.(g=g, V1=time)])
 
 # 'invalid trim argument' with optimization level 1; #1876
```

Comment:
@mattdowle can you take a look here? I added `datatable.optimize=1` to fit the comment and the original issue, which say the fix is related to level-1 optimization, however I didn't see level 1 being applied for the test. And the output says `GForce is on` which doesn't sound right to me.

Reply:
@MichaelChirico  Great investigation. I think probably that because optimization level 2 includes all the optimizations in level 1, that the test didn't reduce the optimization level from 2 to 1. Instead it checked via the `output=` that the optimization that had the issue is on at this point (i.e. level 1 or higher is on).